tests_new/integration_tests/core/organizations/test_organization.py

```diff
@@ -23,160 +23,164 @@ def test_create_organization_with_team_with_permissions(organization1):
 def test_create_organization_with_unauthorized_team(client_noTenantPermissions, group4):
     # Given a user with no tenant permissions to MANAGE ORGANIZATIONS
     # When it creates an organization
-    response = create_organization(client_noTenantPermissions, 'organization2', group4)
-    # Then
-    assert_that(response.errors[0].message).contains('UnauthorizedOperation', 'MANAGE_ORGANIZATIONS')
+    assert_that(create_organization).raises(RuntimeError).when_called_with(
+        client_noTenantPermissions,
+        'organization2',
+        group4,
+    )
```

Comment:
Can we add the `contains...` test that checks the message of the error as we do in lines 62-64?